app/services/importers/create_cohort.rb

```diff
@@ -38,7 +38,6 @@ def create_cohort(row)
           start_year:,
           registration_start_date: safe_parse(row["registration-start-date"]),
           academic_year_start_date: safe_parse(row["academic-year-start-date"]),
-          npq_registration_start_date: safe_parse(row["npq-registration-start-date"]),
```

Comment:
can we please avoid this commit and we can have a different ticket for it? as this is adding too much to an already big pr?